.circleci/config.yml

```diff
@@ -5,12 +5,12 @@ references:
     setup_remote_docker:
       docker_layer_caching: true
     docker:
-      - image: aetrnty/builder
+      - image: aeternity/builder
```

Comment:
Dependent on changes in https://github.com/aeternity/docker-builder ?

Reply:
Not really, but I opened a PR for that repo too.